src/test/resources/database/bag-index.sql

```diff
@@ -2,7 +2,7 @@ create table bag_info (
     bagId text not null primary key,
     base text not null,
     created text not null,
-    doi text not null unique);
+    doi text not null);
```

Comment:
In some cases (some external customers) there is no new DOI given to the new version to be deposited.